src/main/java/com/orbitz/consul/monitoring/ClientEventHandler.java

```diff
@@ -24,18 +22,26 @@ public ClientEventHandler(String clientName, ClientEventCallback callback) {
         this.callback = callback;
     }
 
-    public void httpRequestSuccess(Request request) {
-        EVENT_EXECUTOR.submit(() -> callback.onHttpRequestSuccess(clientName, request.method(), request.url().query()));
+    public void httpRequestSuccess(Request request, Response<?> response) {
+        EVENT_EXECUTOR.submit(() -> {
+            callback.onHttpRequestSuccess(clientName, request.method(), request.url().query());
+            callback.onHttpRequestSuccess(clientName, request.method(), request.url().encodedPath(), request.url().query(), response.code());
```

Comment:
I wonder, is it good to have both callbacks executed?

Maybe it would be better to call the old deprecated method in new default one:
```
default void onHttpRequestSuccess(String clientName, String method, String path, String queryString, int responseCode) {
        // Kept for compatibility at the moment
        onHttpRequestSuccess(clientName, method, queryString);        
}
```
Then replacing calls here would still keep the backwards compatibility?

Reply:
Yes, it's good idea, thank you!